project 2/styles.css

```diff
@@ -0,0 +1,154 @@
+* {
+    box-sizing: border-box;
+}
+
+h1 {
+    font-family: 'Big Shoulders Display', cursive;
```

Comment:
I recommend changing the back-up font here from "cursive" to "sans-serif." I know this is the font stack as provided by Google, but you're free to change it and should.
This is "Big Shoulders Display," the font you chose:
<img width="476" alt="your-font" src="https://user-images.githubusercontent.com/2653980/67909148-95c43800-fb54-11e9-9d49-29d71e8c426a.png">


If that font doesn't load (let's say the Google Fonts server has an outage), "cursive" is the backup font; this is what that looks like:
<img width="467" alt="cursive" src="https://user-images.githubusercontent.com/2653980/67909178-aaa0cb80-fb54-11e9-8328-772a1b7f4918.png">


Really nothing like the font you chose, right?
If you change your font stack to
```suggestion
    font-family: 'Big Shoulders Display', sans-serif;
```


this is what the backup font would look like:
<img width="469" alt="sans-serif" src="https://user-images.githubusercontent.com/2653980/67909196-be4c3200-fb54-11e9-8750-17b3153f9f56.png">


In general, if the backup font listed by Google is either "cursive" or "fantasy," I recommend replacing that with either "serif" or "sans-serif," whichever more closely matches your selected font.